doc_admin/AdminManual.adoc

```diff
@@ -39,83 +48,77 @@ This application software consists of the following parts:
 
 Before starting, check the following software compatibility matrix, in order to verify the external software versions needed to run Open Hospital and the current available extended features for every version, architecture and operating system.
-include::./oh_software_compatibility_matrix.adoc[]
+include::./OH_software_compatibility_matrix.adoc[]
 
 
 [#jre]
 ==== 3.1.2 Java Runtime Environment (JRE)
 
-Depending on the operating system, there are different ways to install a Java Virtual Machine and the Java Runtime Environment (JRE)
-
-For instance, on a Microsoft Windows® system, the JRE is downloaded and setup file is executed, while on Linux based systems a single command is typed into a terminal window.
+Depending on the operating system, there are different ways to install the Java Runtime Environment (JRE).
+The oh.sh / oh.bat scripts can be used to download and install the latest JRE available; it is also possible to install it
+manually, following the specific instructions for the operating system being used.
 
 NOTE: See Software Compatibility Matrix in order to identify the correct Java version.
 
 TIP: If when running Open Hospital the application appears small-scaled or blurry then the issue is probably using a high-dpi display. In such cases there are some solutions (https://stackoverflow.com/a/39372897/5309541[click here]) to apply only on the machines using this type of monitor.
 
-[#mysql-server]
-==== 3.1.3 Database server configuration
+[#database-server]
+==== 3.1.3 Database Server
 
-Depending on the operating system, there are several software tools to install and manage a MySQL server. In most cases just a standard installation package of the software is needed, paying attention to configuration options during the installation process.
+Depending on the operating system, there are several software tools to install and manage a MariaDB/MySQL database server. In most cases just a standard installation package of the software is needed, paying attention to configuration options during the installation process.
 
-For instance, on a Microsoft Windows® system, MySQL Workbench® serves as a client, while on Linux based systems the MySQL Administrator and MySQL Query Browser tools are used to perform the same tasks. Some newer distributions (distros) include MySQL Workbench.
+On a Microsoft Windows® system, MySQL Workbench® can be used as a client to connect to the database server and perform the needed tasks, while on Linux based systems the MySQL Administrator and MySQL Query Browser tools can be used.
 
-The best approach for installation is to use the command-line MySQL client which uses essentially the same commands and syntax for every platform.
+The suggested approach for installation is to use the command-line MariaDB/MySQL client which uses the same commands and syntax for every platform.
 
 NOTE: See Software Compatibility Matrix in order to identify the correct MariaDB / MySQL version.
```

Comment:
Third time is a charm; remove "in order"

Reply:
Done: I've also removed one more "one has to..." :-)